modules/userdirectory/src/main/java/org/opencastproject/userdirectory/UserDirectoryPersistenceUtil.java

```diff
@@ -713,14 +736,15 @@ public static JpaGroup findGroupByRole(String role, String orgId, EntityManagerF
     } catch (NoResultException e) {
       return null;
     } finally {
-      if (em != null)
+      if (em != null) {
         em.close();
+      }
     }
   }
 
 
   public static void removeGroup(String groupId, String orgId, EntityManagerFactory emf) throws NotFoundException,
-  Exception {
+          Exception {
```

Comment:
Isn't that a bit much indentation? Alternatively maybe we can put the whole throws clause into a new line? Same below.

Reply:
Well. When I started this whole adventure, I check how much throw statements were usually indented in Opencast, and the most common indentation was actually 8. So I went with that (and people approved that in the PR that introduced the rules). 

But I agree that this seems weird here. I changed it so that the whole `throws` part with both exception is in a new line. I think it looks better now.